core/src/main/scala/com/avsystem/commons/concurrent/TaskExtensions.scala

```diff
@@ -0,0 +1,57 @@
+package com.avsystem.commons
+package concurrent
+
+import com.avsystem.commons.concurrent.TaskExtensions.{TaskCompanionOps, TaskOps}
+import com.avsystem.commons.misc.Timestamp
+import monix.eval.Task
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.TimeoutException
+import scala.concurrent.duration.FiniteDuration
+
+trait TaskExtensions {
```

Comment:
You seem to have missed committing `TaskExtensionsTest.scala`

Reply:
Tests added